### PR TITLE
Newer version: Microsoft.Sysinternals.ProcessMonitor version 3.95

### LINKED ISSUE #110788: [Update Request]: Microsoft Sysinternals ProcessMonitor
### What type of update are you requesting?

A new version of an existing package

### Current Package Identifier

Microsoft.Sysinternals.ProcessMonitor

### Package Version

3.95

### Please describe the changes you would like to see

There is a new release of ProcessMonitor. 
Documentation: https://learn.microsoft.com/en-us/sysinternals/downloads/procmon

Hashes reported as not matching when trying to install.

### PR DESCRIPTION
**Pull request by BrandonWanHuanSheng**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] This PR only modifies one (1) manifest
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

Resolves #110788 

-----

 ###### Microsoft Reviewers: codeflow:open?pullrequest=https://github.com/microsoft/winget-pkgs/pull/110846&drop=dogfoodAlpha

### CODE REVIEW

**Comment by BrandonWanHuanSheng** [comment]
3.9.4 to 3.9.5 for Procmon update

http://live.sysinternals.com/Procmon.exe New 41984 Bytes for the update
http://live.sysinternals.com/Procmon64.exe New 23000 Bytes for the update

File: manifests/m/Microsoft/Sysinternals/ProcessMonitor/3.9.5/Microsoft.Sysinternals.ProcessMonitor.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# Created using wingetcreate 1.2.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: Microsoft.Sysinternals.ProcessMonitor
+PackageVersion: 3.9.5
```
Comment (mdanish-kh) [inline manifests/m/Microsoft/Sysinternals/ProcessMonitor/3.9.5/Microsoft.Sysinternals.ProcessMonitor.installer.yaml:5]:
```suggestion
PackageVersion: 3.95
```

File: manifests/m/Microsoft/Sysinternals/ProcessMonitor/3.9.5/Microsoft.Sysinternals.ProcessMonitor.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,18 @@
+# Created using wingetcreate 1.2.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: Microsoft.Sysinternals.ProcessMonitor
+PackageVersion: 3.9.5
```
Comment (mdanish-kh) [inline manifests/m/Microsoft/Sysinternals/ProcessMonitor/3.9.5/Microsoft.Sysinternals.ProcessMonitor.locale.en-US.yaml:5]:
```suggestion
PackageVersion: 3.95
```

File: manifests/m/Microsoft/Sysinternals/ProcessMonitor/3.9.5/Microsoft.Sysinternals.ProcessMonitor.yaml [hunk 3]
```diff
@@ -0,0 +1,8 @@
+# Created using wingetcreate 1.2.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.4.0.schema.json
+
+PackageIdentifier: Microsoft.Sysinternals.ProcessMonitor
+PackageVersion: 3.9.5
```
Comment (mdanish-kh) [inline manifests/m/Microsoft/Sysinternals/ProcessMonitor/3.9.5/Microsoft.Sysinternals.ProcessMonitor.yaml:5]:
```suggestion
PackageVersion: 3.95
```

**Review by mdanish-kh — CHANGES_REQUESTED** [review]
Please find the suggestions below. After committing the suggested changes, you would also have to rename the version folder to match the new PackageVersion. The folder under `manifests/m/Microsoft/Sysinternals/ProcessMonitor` should be renamed from `3.9.5` -> `3.95`

A quick and easy way to do this would be here in GitHub Web by pressing the <kbd>.</kbd> (dot) key on your keyboard that'll open up the PR in VS Code Online. Navigate to Explorer menu, find the required folder and make the appropriate rename changes.

After that, stage and commit your changes from Source Control menu on the left. (The changes are automatically pushed)

**Review by mdanish-kh — APPROVED** [review]

**Pull request merged**